test.sh

```diff
@@ -19,29 +19,54 @@ container_name="mroonga_build_test_${timestamp}"
 eval $(grep -E -o '[a-z]+_version=[0-9.]+' $context/Dockerfile)
 mysql_version=$(head -n1 $context/Dockerfile | grep -E -o '[0-9.]{2,}')
 
+function run_sql() {
+  sql=$1
+  docker container exec "${container_name}" mysql -uroot -sse "${sql}"
+}
+
+function assert() {
+  expected=$1
+  actual=$2
+  if [ "$(echo ${expected})" = "$(echo ${actual})" ]; then
+    return 0
+  fi
+  echo -e "Not match.\nexpected: <${expected}>\nactual  : <${actual}>"
+  return 1
+}
+
 sudo docker container run \
   -d \
-  -p 33061:3306 \
   -e MYSQL_ALLOW_EMPTY_PASSWORD=yes \
   --name $container_name \
   $image_name
 ### Should test.
 while true ; do
-  mysqladmin -h 127.0.0.1 -P 33061 -uroot ping && break
+  docker container exec "${container_name}" mysqladmin -uroot ping && break
   sleep 1
 done
-(echo -e "mroonga_libgroonga_version\t${groonga_version}"; \
- echo -e "mroonga_version\t${mroonga_version}"; \
- echo -e "version\t${mysql_version}") \
-  > /tmp/expected.txt
-mysql_e="mysql -h 127.0.0.1 -P 33061 -uroot -sse"
-($mysql_e "SHOW VARIABLES LIKE 'mroonga_libgroonga_version'"; \
- $mysql_e "SHOW VARIABLES LIKE 'mroonga_version'"; \
- $mysql_e "SHOW VARIABLES LIKE 'version'") \
-  > /tmp/actual.txt
+
+for i in {1..30}; do
+  # Need to wait a bit until Mroonga is available.
+  run_sql "SELECT mroonga_command('status')" > /dev/null 2>&1 && break
+  sleep 1
+done
+
 set +e
-diff -u /tmp/expected.txt /tmp/actual.txt
+echo -e "$(run_sql "SELECT JSON_PRETTY(mroonga_command('status'))")" && \
```

Comment:
`echo -e`ってなんで必要なの？`\n`を改行にしたいとか？

```suggestion
run_sql "SELECT JSON_PRETTY(mroonga_command('status'))" && \
```